src/main.c

```diff
@@ -1225,6 +1225,11 @@ accept_mgt_cb(struct ev_loop* loop, struct ev_io* watcher, int revents)
          pgmoneta_management_write_int32(client_fd, ret);
          free(payload_s1);
          break;
+      case MANAGEMENT_ENCRYPT:
+         pgmoneta_log_debug("pgmoneta: Management encrypt: %s/%s", payload_s1);
```

Comment:
There is only one argument - remove `/%s`

Reply:
Please have another look.